src/content/docs/dropins/checkout/containers/estimate-shipping.mdx

```diff
@@ -9,7 +9,7 @@ The `EstimateShipping` container is designed to estimate and display shipping co
 
 ## EstimateShipping configurations
 
-The `EstimateShipping` container is read-only, unlike the editable [`EstimateShipping`](/dropins/cart/cart-slots/#estimateshipping) container in the cart drop-in component. Initially, it displays estimated shipping costs. After a customer provides a shipping address and selects a shipping method, it shows the actual shipping cost. This container is designed to be used as a slot within the `OrderSummary` container from the cart, where the estimated shipping information is displayed.
+The `EstimateShipping` container is read-only, unlike the editable `EstimateShipping` container in the cart drop-in component. Initially, it displays estimated shipping costs. After a customer provides a shipping address and selects a shipping method, it shows the actual shipping cost. This container is designed to be used as a slot within the `OrderSummary` container from the cart, where the estimated shipping information is displayed.
```

Comment:
Does the cart still have an EstimateShipping container? Maybe we need to remove the second part of the first sentence.